CONTRIBUTING.md

```diff
@@ -1,30 +1,30 @@
-If you want to contribute to Theano, have a look at the instructions here:
-http://deeplearning.net/software/theano/dev_start_guide.html
+If you want to contribute to Aesara, have a look at the instructions here:
+https://aesara.readthedocs.io/en/latest/dev_start_guide.html
 
 
-## Migrating PRs from original Theano Repo
-Theano-PyMC is actively merging new changes. If you have a pull request on the original respository and would like to move it here use the following commands in your local theano repo
+## Migrating PRs from original Aesara Repo
```

Comment:
I think this section should either be removed altogether or keep the theano

Reply:
Yes, definitely.